pwnlib/util/proc.py

```diff
@@ -94,7 +94,7 @@ def match(p):
 
     processes = sorted(processes, key=lambda p: p.create_time())
 
-    return reversed([p.pid for p in processes])
+    return list(reversed([p.pid for p in processes]))
```

Comment:
Then you can go with just:

```suggestion
    return [p.pid for p in processes]
```